scripts.py

```diff
@@ -345,22 +345,6 @@ def monitors(self, **kwargs):
         return [cls(self._db, **kwargs) for cls in ReaperMonitor.REGISTRY]
 
 
-class UpdateSearchIndexScript(RunMonitorScript):
-
-    def __init__(self):
-        parser = argparse.ArgumentParser()
-        parser.add_argument(
-            '--works-index',
-            help='The ElasticSearch index to update, if other than the default.'
-        )
```

Comment:
I don't think we need the `--works-index` argument. If you use a different index for works, you'll configure it as part of your search integration, and we'll pick it up when we instantiate `ExternalSearchIndex`.

Reply:
I think the original use case was to build a new index while still using the old index during an update.